schema.graphql

```diff
@@ -169,11 +176,140 @@ type CustomerAddress {
   vatNumber: String
 }
 
+type Package {
+  "The invoice number for this package"
+  invoiceNumber: Number
+
+  "The Track and Trace number for this package"
+  trackAndTraceNumber: String
+}
+
 enum CustomerAddressType {
   INVOICE
   DELIVERY
 }
 
+enum OrderStatus {
+  CREATED
+  PROFORMA_INVOICE
+  READY_TO_BE_SENT_TO_SUPPLIER
+  SENT_TO_THE_SUPPLIER
+  PARTIALLY_RECEIVED
+  READY_FOR_PARTIAL_DELIVERY
+  RECEIVED
+  READY_FOR_DELIVERY
+  SENT_TO_CUSTOMER
+}
+
+enum ShipmentType {
+  SHIPMENT_TO_CUSTOMER_ADDRESS
+  PICKUP_FROM_POSTNL_LOCATION
+  PICKUP_ORDER
+}
+
+enum PaymentMethodType {
+  CASH_ON_DELIVERY
+  ON_ACCOUNT
+  IN_ADVANCE_BY_BANK
+  CONTANT
+  CREDIT
+  IDEAL
+  CREDIT_CARD
+  CREDIT_APPROVAL
+  SOFORT
+  KLARNA
+}
+
+type OrderItem {
+  "Internal number (User-visible, sku in Magento)"
+  internalNumber: String!
```

Comment:
Dit is eigenlijk niet de SKU, maar willen we evt wel tonen aan de gebruiker. Het ziet er wel uit als een SKU, maar is bij KOmmaGo niet uniek, dus schiet KommaGo hun database ID in als SKU (net als bij Verdouw)

Dus ik denk dat we dit veld wel kunnen houden (evt comment aanpassen), en er een tweede veld, ID moet bijkomen.

Reply:
Ik heb het ID veld toegevoegd.